bukuserver/translations/messages_custom.pot

```diff
@@ -0,0 +1,53 @@
+msgid "Original and replacement tags are the same."
+msgstr ""
```

Comment:
These strings can't be detected so I'm appending them after template generation (e.g. exception messages or generated text)